features/enable_fips_vm.feature

```diff
@@ -175,256 +171,44 @@ Feature: FIPS enablement in lxd VMs
       A reboot is required to complete disable operation
       """
     When I reboot the machine
-    Then I verify that `openssh-server` installed version matches regexp `fips`
-    And I verify that `openssh-client` installed version matches regexp `fips`
-    And I verify that `strongswan` installed version matches regexp `fips`
-    And I verify that `openssh-server-hmac` installed version matches regexp `fips`
-    And I verify that `openssh-client-hmac` installed version matches regexp `fips`
-    And I verify that `strongswan-hmac` installed version matches regexp `fips`
-    When I run `apt-mark unhold openssh-client openssh-server strongswan` with sudo
-    Then I will see the following on stdout:
-      """
-      openssh-client was already not hold.
-      openssh-server was already not hold.
-      strongswan was already not hold.
-      """
-    And I verify that `<fips-service>` is disabled
+    Then I verify that packages `<fips-packages>` installed versions match regexp `<fips-regex>`
+    And I verify that `fips-updates` is disabled
     When I verify that running `pro enable fips --assume-yes` `with sudo` exits `1`
     Then stdout matches regexp:
       """
       Cannot enable FIPS because FIPS Updates was once enabled.
       """
     And I verify that files exist matching `/var/lib/ubuntu-advantage/services-once-enabled`
-    When I run `pro enable <fips-service> --assume-yes` with sudo
+    When I run `pro enable fips-updates --assume-yes` with sudo
     And I reboot the machine
-    Then I verify that `<fips-service>` is enabled
-    And I verify that `livepatch` is disabled
+    Then I verify that `fips-updates` is enabled
+    When I run `pro disable livepatch` with sudo
+    Then I verify that `livepatch` is disabled
     When I run `pro enable livepatch --assume-yes` with sudo
-    Then I verify that `<fips-service>` is enabled
+    Then I verify that `fips-updates` is enabled
     And I verify that `livepatch` is enabled
-    When I run `pro status --all --format json` with sudo
-    Then stdout contains substring:
-      """
-      {"available": "no", "blocked_by": [{"name": "livepatch", "reason": "Livepatch cannot be enabled while running the official FIPS certified kernel. If you would like a FIPS compliant kernel with additional bug fixes and security updates, you can use the FIPS Updates service with Livepatch.", "reason_code": "livepatch-invalidates-fips"}, {"name": "fips-updates", "reason": "FIPS cannot be enabled if FIPS Updates has ever been enabled because FIPS Updates installs security patches that aren't officially certified.", "reason_code": "fips-updates-invalidates-fips"}], "description": "NIST-certified FIPS crypto packages", "description_override": null, "entitled": "yes", "name": "fips", "status": "n/a", "status_details": "Cannot enable FIPS when FIPS Updates is enabled.", "warning": null}
```

Comment:
I think we lost this check

Reply:
I removed this one because we already have a step where we try to enable fips and we see the message:
```
Cannot enable FIPS because FIPS Updates was once enabled.
```

That's why I considered this redundant, but let me know if you disagree